docs/samples/Microsoft.ML.GenAI.Samples/Program.cs

```diff
@@ -1,4 +1,6 @@
 ﻿// See https://aka.ms/new-console-template for more information
+using Microsoft.ML.GenAI.Samples.Llama;
 using Microsoft.ML.GenAI.Samples.Phi3Mini;
 
-await SemanticKernelSample.RunChatCompletionSample();
+LlamaSample.Run();
+//await AutoGenSample.RunAsync();
```

Comment:
If we don't need the commented code lets remove it.

Reply:
Good point, I'm going to revert this change.